pycti/entities/opencti_identity.py

```diff
@@ -355,7 +355,23 @@ def create(self, **kwargs):
                 """
                 input_variables["x_opencti_firstname"] = x_opencti_firstname
                 input_variables["x_opencti_lastname"] = x_opencti_lastname
+                input_variables["x_opencti_reliability"] = x_opencti_reliability
                 result_data_field = "individualAdd"
+            elif type == IdentityTypes.SYSTEM.value:
```

Comment:
Can you add a test on it ? (in tests/02-integration/entities)
Thank you

Reply:
Simple test added